### DISCUSSION TITLE
How to use it ?

### DISCUSSION THREAD

**Discussion by tonycstech** [THUDM/CogVideo#488] [Q&A] [answered]
I dont see any release to download. How does it work ?

**Comment by vivekvp1** [comment]
Try using [Pinokio] https://[pinokio](https://pinokio.computer/).computer/

**Reply by tonycstech** [reply]
that seems to work but it does not seem to do anything. The CogStudio i mean. I do not see any GPU activity when its generating.
I am not even sure its generating anything at this point.

**Reply by senryusama** [reply]
> Try using [Pinokio] https://[pinokio](https://pinokio.computer/).computer/

Tried that, it downloaded everything to start CogVideo, but after hitting the "generate video" the first time, it started to download a TON of stuff....it's already at 70-80 gb downloaded and it keeps going on....WTH? Maybe that's why @tonycstech tought it wasn't working.

**Reply by tonycstech** [reply]
you gotta go through lots of downloads 1st.
I downloaded about 80 gigs too.
1st time it didnt complete the download 100% so things were still missing.
Also dont jump from tab to tab., just focus on one tab and wait until results are posted.
Will take a while.

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
This is an open-source model that should run on diffusers, and our readme mentions how to download the model from Huggingface and use our open-source model in the diffusers repository of transformrer.
